### ISSUE TITLE
add `--ignore-missing` flag

### ISSUE THREAD

**Issue by georgettica** [rhash/RHash#180]
similar tools in this realm `sha256sum` or `shasum` have and `--ignore-missing` which allows verifying only one binary if a list of them are displayed in the checksum

I would love for this flag to exist and return a 0 exit code even if not all of the files are not found

I can handle the case that the file didn't download correctly and is missing to run the check aka:

```
curl -sSLo file.tar.gz https://path/to/file.tar.gz-bad
curl -sSLo all-checksums.txt https://path/to/all-checksums.txt
rhas -c all-checksums.txt -a
ALL-OK
```

**Comment by georgettica** [comment]
see the issue here to see my use case
https://github.com/openshift/ocm-container/issues/83

**Comment by rhash** [comment]
Supported by commit 568a5421e8bdebaba3d1c8ec134fdaa963dd11d7.

**Comment by rhash** [comment]
The feature got included into [RHash v1.4.3](https://github.com/rhash/RHash/releases/tag/v1.4.3) release.
